frontend/cypress/integration/LibraryTest.js

```diff
@@ -0,0 +1,40 @@
+import baseVisit from '../helper';
+
+describe('This test checks the rendered content' , () => {
+    it('checks the whole page', () => {
+      baseVisit('/library');
+    });
+
+    it('Checks scan your book barcode card' , () => {
+        cy.get('[data-cy=FirstCard]')
+        cy.contains('Scan your book barcode')
+        cy.contains('Bokens Streckkod')
+        cy.contains('Return')  
+        
+    })
+
+    it('Checks scan the book barcode card' , () => {
+        cy.get('[data-cy=SecondCard]')
+        cy.contains('Scan the book barcode')
+        cy.contains('Bokens Streckkod')
+        cy.contains('Låna Ut')    
+    })
+    
+    it('Checks the data tables' , () => {
+        cy.get('[data-cy=DataTable]')
```

Comment:
```suggestion
        cy.get('[data-cy=table]')
```

Here we can use the `data-cy` tag that has already been defined.